libfreefare/freefare.3

```diff
@@ -117,6 +120,9 @@ can be gathered using the
 and
 .Fn freefare_get_tag_friendly_name
 functions.
+.Pp
+.Fn freefare_set_tag_timeout
+allows to set the timeout for a given NFC operation. Default value is 500ms.
```

Comment:
The function description should describe what value is expected for `timeout`, e.g.

```roff
The
.Fn freefare_set_tag_timeout
function is used to set the maximum duration for NFC operations to
.Fa timeout
mili-seconds.  Setting
.Fa timeout
to 0 disable the timeout feature.  By default, no timeout is configured.
```

Reply:
Documentation updated as well as Makefile.am